_non-cosmos/ethereum/assetlist.json

```diff
@@ -1382,6 +1382,237 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/paxg.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/paxg.svg"
       }
+    }, 
+    {
+      "description": "CRV is the governance token for Curve Finance.",
+      "type_asset": "erc20",
+      "address": "0xd533a949740bb3306d119cc777fa900ba034cd52",
+      "denom_units": [
+        {
+          "denom": "0xd533a949740bb3306d119cc777fa900ba034cd52",
+          "exponent": 0,
+          "aliases": [
+            "crv-wei"
+          ]
+        },
+        {
+          "denom": "crv",
+          "exponent": 18
+        }
+      ],
+      "base": "0xd533a949740bb3306d119cc777fa900ba034cd52",
+      "name": "Curve DAO",
+      "display": "crv",
+      "symbol": "CRV",
+      "coingecko_id": "curve-dao-token",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/crv.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/crv.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/crv.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/crv.svg"
+      }
+    },
+    {
+      "description": "A collateralized-debt-position (CDP) stablecoin by Curve DAO.",
+      "type_asset": "erc20",
+      "address": "0xf939e0a03fb07f59a73314e73794be0e57ac1b4e",
+      "denom_units": [
+        {
+          "denom": "0xf939e0a03fb07f59a73314e73794be0e57ac1b4e",
+          "exponent": 0,
+          "aliases": [
+            "crvusd-wei"
+          ]
+        },
+        {
+          "denom": "crvusd",
+          "exponent": 18
+        }
+      ],
+      "base": "0xf939e0a03fb07f59a73314e73794be0e57ac1b4e",
+      "name": "crvUSD",
+      "display": "crvusd",
+      "symbol": "crvUSD",
+      "coingecko_id": "crvusd",
+      "images": [
```

Comment:
please define trace for crvUSD as a USD synth

Reply:
"traces": {
  "type": "synthetic",
  "counterparty": {
    "chain_name": "forex",
    "base_denom": "USD"
  },
  "provider": "Curve Finance"
},